rs/nns/governance/src/neuron/types.rs

```diff
@@ -348,14 +353,57 @@ impl Neuron {
             > 0
     }
 
+    fn deciding_voting_power_adjustment_factor(
```

Comment:
Please add a doc string to explain what this function returns, ideally with an example. 

Mention that it's result in between 0 and 1.